src/hps_align/plot/kinks.py

```diff
@@ -0,0 +1,136 @@
+import ROOT as r
+from ._plotter import Plotter
+from . import alignment_utils
+from .index_page import htmlWriter
+
+
+@Plotter.user
+def Lambda(p: Plotter):
```

Comment:
any reason why lambda is capitalized but phi isn't?

Reply:
In Python, `lambda` is a reserved word to use to define "anonymous" functions. In order to avoid any conflict, I had to capitalize `L`. I'm open to other ways to avoid `lambda` (e.g. maybe just call this `tan_lambda`?).

Reply:
yeah, let's do `tan_lambda`!